packages/lisk-p2p/src/peer_selector.ts

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import { NotEnoughPeersError } from './errors';
+import { Peer } from './peer';
+
+// For future extension of return type
+export interface PeerReturnType {
```

Comment:
Do we need `PeerReturnType`? Maybe we should return `ReadonlyArray<Peer>` from the selectPeers method instead to keep it simple.

Reply:
I thought it will be easier to add any other field if we want to return something else from this function that can be added in future but I guess it could be a simple `ReadonlyArray<Peer>`